src/main/java/core/basesyntax/Lotery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lotery {
+    public static int getRandomBall() {
+        Random randBall = new Random();
+        int num = randBall.nextInt(100);
```

Comment:
100 should be made a constant field (private static final field with name in UPPER_CASE)
https://www.javatpoint.com/java-constant